CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ### Version 9.1
 * Allows query parameters to match on a substring. Ex `q=body:{body}`
+* Support for full path in `LoadBalancingTarget`.
```

Comment:
ps it is best to keep details to a minimum in the change log. Details are good for your actual commit message.

Ex. for the changelog:

* Supports context path when using Ribbon `LoadBalancingTarget`

For the commit message:
```
Supports context path when using Ribbon LoadBalancingTarget

Before, `LoadBalancingTarget` stripped out the path and only used
`URI.getScheme()` and `URI.getHost()` to generate the `Request`.

Now, it will add `URI.getPath()` to the `Request` as well; this is
useful if you want to interact with endpoints with a context-path.
```